waterloop-site/src/components/SustainableTech/NavBar/NavBar.tsx

```diff
@@ -1,5 +1,141 @@
 import React from 'react';
+import styled from 'styled-components';
+import IconBlackImg from '../../../static/img/logos/Icon_Black.svg';
+import { NavLink } from 'react-router-dom';
+import scrollTo from '../utils/LinkScroll';
 
-const NavBar: React.FC = () => <div>STE NavBar</div>;
+const NavbarContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 64px;
+  background-color: #e5f6fa;
+  position: fixed;
+  top: 0;
+  scroll-behavior: smooth;
+`;
 
+const IconBlack = styled.img`
+  width: 50px;
+  height: 50px;
+  -webkit-transition: 0.2s ease-in-out;
+  transition: 0.2s ease-in-out;
+
+  @media screen and (max-width: 900px) {
+    width: 40px;
+    height: 40px;
+  }
+
+  @media screen and (max-width: 600px) {
+    width: 30px;
+    height: 30px;
+  }
+`;
+
+const LogoContainer = styled.div`
+  padding-left: 4vw;
+`;
+
+const ListContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: nowrap;
+  align-items: center;
+`;
+
+const ScrollLink = styled.button`
+  color: black;
+  margin: 0 15pt;
```

Comment:
Is there a specific reason you chose to use `pt`? Generally our team uses `px`, `em`, `rem`, `%`